draft-bbs-signatures.md

```diff
@@ -523,20 +520,27 @@ Parameters:
 
 Definitions:
 
-- L, is the non-negative integer representing the number of messages to be signed e.g length(msg_1,...,msg_L). Note if no messages are supplied as an input to this operation, the value of L MUST evaluate to zero (0).
-- R, is the non-negative integer representing the number of revealed messages e.g length(RevealedIndexes). Note if no revealed messages are supplied as an input to this operation, the value of R MUST evaluate to zero (0).
+- L, is the non-negative integer representing the number of signed messages e.g., length([(H_1, msg_1, revealed_1), ..., (H_L, msg_L, revealed_L)]). Note if no messages are signed, the value of L MUST evaluate to zero (0).
+- R, is the non-negative integer representing the number of revealed messages e.g., R = revealed_1 + revealed_2 + ... + revealed_L. If no revealed messages are supplied as an input to this operation, the value of R MUST evaluate to zero (0).
```

Comment:
```suggestion
- R, is the non-negative integer representing the number of revealed messages e.g., R = revealed_1 + revealed_2 + ... + revealed_L. In the calculation of R its assumed that the boolean value of true evaluates to 1 whereas the boolean value of false evaluates to 0.  If no messages are supplied as an input to this operation, the value of R MUST evaluate to zero (0).
```

Reply:
Trying to clarify that effectively we are casting boolean values in the calculation of R to an integer?

Reply:
Good point! Updated to a different wording that it is sorter but may not be as clear?? If yes i will update to the suggestion.